### ISSUE TITLE
Open Chatgpt-web display 500 Server error

### ISSUE THREAD

**Issue by actele** [Kerwin1202/chatgpt-web#360]
Open Chatgpt-web display 500 Server error.

**Comment by Kerwin1202** [comment]
Please provide your deployment method, and if it is Docker, please provide the Docker logs.

**Comment by actele** [comment]
Yes, using Docker, the issue is the same as https://chatgpt-web-kappa-inky.vercel.app

**Comment by actele** [comment]
thx!

**Comment by 02000823** [comment]
在使用docker部署，使用谷歌浏览器打开的情况下经常会出现500的错误，但是在清空浏览器的浏览数据后又恢复正常
![333](https://github.com/Kerwin1202/chatgpt-web/assets/84962546/be3460a3-abc1-45ce-9b1e-41c88e5d3a72)

**Comment by Kerwin1202** [comment]
@02000823 f12 看下 应该是某个接口报错，把相关信息发下，然后看下对应的docker log 是否正常呢

**Comment by 02000823** [comment]
![image](https://github.com/Kerwin1202/chatgpt-web/assets/84962546/0bf5c0a9-8ab5-4775-b277-085c4b9f2b57)

**Comment by biaozhang64** [comment]
> 在使用docker部署，使用谷歌浏览器打开的情况下经常会出现500的错误，但是在清空浏览器的浏览数据后又恢复正常.使用edge浏览器就不会经常出现这种情况 ![333](https://user-images.githubusercontent.com/84962546/280909989-be3460a3-abc1-45ce-9b1e-41c88e5d3a72.png)

我也遇到过这种情况，清除下浏览器缓存就好了
